lib/Service/SettingsService.php

```diff
@@ -41,7 +43,7 @@ public function __construct(
 			'fileSuffix' => $this->getListAttrs('fileSuffix', [...$this->defaultSuffixes, 'custom']),
 			'notesPath' => [
 				'default' => function (string $uid) {
-					return $this->getDefaultNotesPath($uid);
+					return $this->getDefaultNotesNode($uid)['path'];
```

Comment:
not the biggest fan of that, I wonder if it would make sense to keep the old getDefaultNotesPath as it is for this part